parity/run.rs

```diff
@@ -286,7 +276,7 @@ fn execute_light_impl<Cr>(cmd: RunCmd, logger: Arc<RotatingLogger>, on_client_rq
 		network_id: cmd.network_id.unwrap_or(spec.network_id()),
 		subprotocol_name: sync::LIGHT_PROTOCOL,
 		handlers: vec![on_demand.clone()],
-		attached_protos: attached_protos,
+		attached_protos: Vec::new(),
```

Comment:
is it used at all?

Reply:
Doesn't seem so. Good catch. @debris remove it?

Reply:
done